### PR TITLE
Signup with gooogle to Login with Google

### LINKED ISSUE #138: bug : [Button name is login but shows sign  up make it both login and sign up]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**
image

**Additional context**

### PR DESCRIPTION
**Pull request by RamakrushnaBiswal**
**What kind of change does this PR introduce?**
Signup with gooogle to Login with Google

**Issue Number:**

Fixes #138 

**Snapshots/Videos:**
image

**Other information**

### CODE REVIEW

File: src/components/Signup/Signup.tsx [hunk 1]
```diff
@@ -97,10 +97,10 @@ const Signup: React.FC<SignupProps> = ({ isShow, returnShow }) => {
     <>
       <Modal show={show} onHide={handleClose}>
         <Modal.Header closeButton>
-          <Modal.Title>Sign Up</Modal.Title>
+          <Modal.Title>Log In</Modal.Title>
```
Comment (Doraemon012) [inline src/components/Signup/Signup.tsx:100]: Make it `Login or Sign Up`

File: src/components/Signup/Signup.tsx [hunk 2]
```diff
@@ -97,10 +97,10 @@ const Signup: React.FC<SignupProps> = ({ isShow, returnShow }) => {
...
         </Modal.Header>
         <Modal.Body className="d-flex justify-content-center align-items-center m-5">
-          <GoogleButton type="light" onClick={logGoogleUser} />
+          <GoogleButton type="light" onClick={logGoogleUser} label="Login with Google"/>
```
Comment (Doraemon012) [inline src/components/Signup/Signup.tsx:103]: @RamakrushnaBiswal Will the user always sign up and never log in ?
You just had to add this `label` prop. 
Make this `Continue with Google`

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Make the requested changes.

File: src/components/Signup/Signup.tsx (hunk 2 above)
Comment (RamakrushnaBiswal) [reply src/components/Signup/Signup.tsx:103]: Okay

File: src/components/Signup/Signup.tsx (hunk 1 above)
Comment (RamakrushnaBiswal) [reply src/components/Signup/Signup.tsx:100]: Okay

File: src/components/Signup/Signup.tsx (hunk 2 above)
Comment (RamakrushnaBiswal) [reply src/components/Signup/Signup.tsx]: completed  [833f61e](https://github.com/Tanay-ErrorCode/lupo-skill/pull/146/commits/833f61e1f5396f23b4aa4fd4f8078698688b5d13)

File: src/components/Signup/Signup.tsx (hunk 1 above)
Comment (RamakrushnaBiswal) [reply src/components/Signup/Signup.tsx]: completed [833f61e](https://github.com/Tanay-ErrorCode/lupo-skill/pull/146/commits/833f61e1f5396f23b4aa4fd4f8078698688b5d13)

**Review by Doraemon012 — APPROVED** [review]
LGTM !

**Comment by Tanay-ErrorCode** [comment]
@RamakrushnaBiswal Good work!

**Comment by RamakrushnaBiswal** [comment]
@Tanay-ErrorCode add some labels like 1,2,3.

**Pull request merged**